provisioning/tasks/certificates/generate_certificate.yml

```
@@ -0,0 +1,23 @@
+---
+- stat:
+    path: "{{ location }}"
+  register: certificate_folder
+
+- name: Create certificates
+  command: "mkcert {{ host }}"
+  when: not certificate_folder.stat.exists
+
+- name: Creates certificate directories
+  file:
+    path: "{{ location }}"
+    state: directory
+    recurse: yes
+  when: not certificate_folder.stat.exists
+
+- name: Move certificates
+  command: "mv ./{{ host }}.pem {{ location }}"
```

```suggestion
  command: "mv ./{{ host }}.pem {{ location }}"
```

copy mit remote_src, ggf danach die quelldatei löschen